login.go

```diff
@@ -165,7 +165,13 @@ func (cli *Client) Login(ctx context.Context, email, password string) (string, e
 	return response.Token, nil
 }
 
-// Login2FARequest is the request body for the Login 2FA API.
+// Login2FAMethod is a container for the 2FA provider and code used during login.
+type Login2FAMethod struct {
```

Comment:
Since this overlaps the Login2FARequest, that might be reused - but that felt incorrect in intent.